.github/workflows/markdown.links.config.json

```diff
@@ -0,0 +1,13 @@
+{
+    "replacementPatterns": [
+      {
+        "pattern": "^/",
+        "replacement": "https://keptn-experimental-docs-site.netlify.app/"
```

Comment:
Why are we doing this?

:memo: https://github.com/tcort/markdown-link-check#config-file-format

Reply:
Why are we doing this ^ @iamrajiv ?

Reply:
After discussion with @iamrajiv , he's going to try this out ^ and show us the results here with a before and after (replacement) in the docs.